Sources/MongoSwift/ChangeStreamOptions.swift

```diff
@@ -56,7 +56,7 @@ public struct ChangeStreamOptions: Codable {
      * - SeeAlso: https://docs.mongodb.com/master/changeStreams/#change-stream-start-after
      */
     // TODO: SWIFT-519 - Make this public when support is added for 4.2 change stream features.
-    internal var startAfter: ResumeToken?
+    // internal var startAfter: ResumeToken?
```

Comment:
I commented this out because I use `Mirror` elsewhere to determine the valid arguments for `createChangeStream`, and having this property here could inadvertently make it seem like we support `startAfter`.